index.html

```diff
@@ -1,15 +1,138 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     <title>Haiti Tech Submit</title>
+    <link rel="shortcut icon" href="./public/images/logo.jpg" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://use.typekit.net/dli5exx.css">
     <link rel="stylesheet" href="./public/css/style.css">
 </head>
+
 <body>
-    
+    <!-- Navigation bar -->
+    <header class="header">
+        <div class="social-nav">
+            <div class="second-menu">
+                <ul>
+                    <li><a href=""><i class="fa fa-facebook-f"></i></a></li>
+                    <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Logout</a></li>
+                </ul>
+            </div>
+
+        </div>
+        <!-- Logo -->
+        <a href="./index.html" class="logo">
+            <img src="./public/images/logo.jpg" alt="Logo of HTS">
+        </a>
+
+        <!-- Hamburger icon -->
+        <input class="side-menu" type="checkbox" id="side-menu" />
+        <label class="hamb" for="side-menu"><span class="hamb-line"></span></label>
+
+        <!-- Menu -->
+
+        <nav class="nav">
+            <ul class="menu">
+                <li><a href="./about.html">About</a></li></a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#">HTS Campain</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Hello, I would advise you to follow what the previous reviewer requested: The background design of the Mobile is `transparent` or the `same color`/`pattern` as the top section background.

- [ ]  Look at the pictures below for a better understanding.:
- **Your design:** Can you see two different colors?
![Capture](https://user-images.githubusercontent.com/63972670/234873330-0d74392e-20b1-42ab-9899-38ff1276aad0.JPG)
- Original design:
![Capture](https://user-images.githubusercontent.com/63972670/234873748-f5dc8b4f-47cb-4cd7-acc0-a1634507ac09.JPG)
NB: I am requesting this because the project requirement recommends the original design.